<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormRulesDialog</class>
 <widget class="QDialog" name="FormRulesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>300</width>
    <height>300</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>300</width>
    <height>300</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Add Rule Block</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox_fromRulesDialog">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>260</y>
     <width>190</width>
     <height>30</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Orientation::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::StandardButton::No|QDialogButtonBox::StandardButton::Yes</set>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>281</width>
     <height>241</height>
    </rect>
   </property>
   <property name="title">
    <string>Information About Blocking Rules</string>
   </property>
   <widget class="QLabel" name="label_IP">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>IPv4</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Port">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>80</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Port</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Protocol">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Protocol</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Name">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Name</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_Direction">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>50</y>
      <width>50</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Direction</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_setName">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>20</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_setDirection">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>50</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_setPort">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>80</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_setProtocol">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>110</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_setIP">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>140</y>
      <width>200</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Null</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_fromRulesDialog</sender>
   <signal>accepted()</signal>
   <receiver>FormRulesDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_fromRulesDialog</sender>
   <signal>rejected()</signal>
   <receiver>FormRulesDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
